bigbio/hub/hub_repos/symptemist/symptemist.py

```diff
@@ -0,0 +1,291 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from .bigbiohub import kb_features
+from .bigbiohub import BigBioConfig
+from .bigbiohub import Tasks
+
+_LANGUAGES = ["Spanish"]
+_PUBMED = False
+_LOCAL = False
+_CITATION = """\
+@inproceedings{lima2023overview,
+  title={Overview of SympTEMIST at BioCreative VIII: corpus, guidelines and evaluation of systems for the detection and normalization of symptoms, signs and findings from text},
+  author={Lima-L{\'o}pez, Salvador and Farr{\'e}-Maduell, Eul{\`a}lia and Gasco-S{\'a}nchez, Luis and Rodr{\'\i}guez-Miret, Jan and Krallinger, Martin},
+  booktitle={Proceedings of the BioCreative VIII Challenge and Workshop: Curation and Evaluation in the era of Generative Models},
+  year={2023}
+}
+"""
+
+_DATASETNAME = "symptemist"
+_DISPLAYNAME = "SympTEMIST"
+
+_DESCRIPTION = """\
+The SympTEMIST corpus is a collection of 1,000 clinical case reports in Spanish annotated with symptoms, signs and findings mentions and normalized to SNOMED CT. The texts belong to the SPACCC corpus and are the same ones used in SympTEMIST and MedProcNER, making the annotations complementary for medical entity recognition.
+"""
+
+_HOMEPAGE = "https://temu.bsc.es/symptemist/"
+
+_LICENSE = "CC_BY_4p0"
+
+_URLS = {
+    _DATASETNAME: "https://zenodo.org/records/10635215/files/symptemist-complete_240208.zip?download=1",
+}
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION, Tasks.NAMED_ENTITY_DISAMBIGUATION]
```

Comment:
`NAMED_ENTITY_DISAMBIGUATION` is in supported tasks, but the tests report that there are no normalization entries (see stacktrace below). @phlobo is this intended?

```
======================================================================
FAIL: runTest (__main__.TestDataLoader.runTest) [Check schema validity]
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/lwg/projects/biomedical/tests/test_bigbio_hub.py", line 130, in runTest
    self.test_schema(schema)
  File "/Users/lwg/projects/biomedical/tests/test_bigbio_hub.py", line 761, in test_schema
    raise AssertionError(f"Required key '{non_empty_feature}' does not have any instances")
AssertionError: Required key 'normalized' does not have any instances
```

Reply:
There were multiple tracks (one with entities only and another one with normalization). We had the same issue with DisTEMIST: https://github.com/bigscience-workshop/biomedical/pull/736#discussion_r934932383

Reply:
Thanks for clarifying!